layouts/default.vue

```diff
@@ -99,7 +109,23 @@
     },
 
     watch: {
-      '$i18n.locale': '$fetch'
+      '$i18n.locale': '$fetch',
+      $route(to, from) {
+        this.$nextTick(() => {
+          if (to.path === from.path) {
+            this.enableAnnouncer = false;
+          } else {
+            this.$refs.resetfocus.setAttribute('tabindex', '0');
+            this.$refs.resetfocus.focus();
+            this.enableAnnouncer = true;
+          }
+        });
+      }
+
+    },
+
+    mounted() {
+      this.$announcer.setComplementRoute(this.$t('pageHasLoaded'));
```

Comment:
This should not be necessary, because `fetch` is a Nuxt lifecycle function called when the page is first rendered, as well as by the watching of `$i18n.locale`